### PR TITLE
DataGrid.ExcelJS export - thrown the deprecate warning when the 'cell' field of customizeCell handler argument is used

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by jsdmitry — DISMISSED** [review]

**Review by novsstation — DISMISSED** [review]

File: testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js [hunk 1]
```diff
@@ -6477,6 +6478,72 @@ QUnit.module('_getFullOptions', moduleConfig, () => {
     });
 });
 
+QUnit.module('Deprecated warnings', moduleConfig, () => {
+    QUnit.test('CustomizeCell handler - warnings when \'cell\' field is not used', function(assert) {
+        assert.expect(3);
+        const stub = sinon.stub(errors, 'log', () => {
+            assert.strictEqual(true, false, 'error.log should not be called');
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js]: IMHO we need this 'assert' in each test in its 'after' function:
https://github.com/DevExpress/DevExtreme/blob/20_2/testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js#L54 - let's test the latest API only and pray that there are no issues in obsoleted

File: testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js [hunk 2]
```diff
@@ -6477,6 +6478,72 @@ QUnit.module('_getFullOptions', moduleConfig, () => {
...
+        });
+
+        const done = assert.async();
+        const ds = [{ f1: 'f1_1' }];
+
+        const dataGrid = $('#dataGrid').dxDataGrid({
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js]: testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js - will we rename this file to something like '\dataGrid.exceljsParts\' or 'exceljs.datagrid.tests.js' ?

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js (hunk 2 above)
Comment (EugeniyKiyashko) [reply testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js]: Yes, we will rename this file in the future. We are going to create a separate commit for this purpose

Comment (EugeniyKiyashko) [reply testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js]: I renamed it

File: testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js]: Fixed

**Review by novsstation — APPROVED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**